src/inventory.go

```diff
@@ -19,6 +29,10 @@ func getInventory() (map[string]interface{}, error) {
 	i := make(inventory.Item)
 	err = yaml.Unmarshal(rawYamlFile, &i)
 
+	if len(i) == 0 {
```

Comment:
should we check the err above?

Right now it is weird that we check the # of elements before the err returned by Unmarshal

Reply:
thanks! good catch!